README.md

```diff
@@ -79,11 +79,11 @@ To see all available Jupyter Notebook commands, open the Command Palette and typ
 
 Learn more about the rich features of the Jupyter extension:
 
--   [IntelliSense](https://code.visualstudio.com/docs/python/editing#_autocomplete-and-intellisense): Edit your code with auto-completion, code navigation, syntax checking and more!
+-   [IntelliSense](https://code.visualstudio.com/docs/python/editing#_autocomplete-and-intellisense): Edit your code with auto-completion, code navigation, syntax checking and more.
 
     -   _May be limited due to kernelspec of choice_
```

Comment:
This disclaimer seems unnecessary, even though it's true. I mean intellisense is also limited to the language you're working with and we're not disclaiming that.